src/Controls/src/Core/VisualElement/VisualElement.Platform.cs

```diff
@@ -41,7 +41,10 @@ partial void HandlePlatformUnloadedLoaded()
 					// If SendUnloaded caused the unloaded tokens to wire up
 					_loadedUnloadedToken?.Dispose();
 					_loadedUnloadedToken = null;
-					_loadedUnloadedToken = this.OnLoaded(SendLoaded);
+					if (Handler is not null)
```

Comment:
Can't this be consumed on the Controls and not the handler? So this should fire even if theres no handler because you can subscribe to it for example on a ctor of a ContentView no @PureWeen  ?